sklearn/kernel_ridge/tests/test_krc.py

```diff
@@ -0,0 +1,203 @@
+"""
+Testing for Kernel Ridge Classification
+(sklearn.kernel_ridge)
+
+Authors: Carlos Perales <sir.perales@gmail.com>
+License: BSD 3 clause
+
+"""
+import numpy as np
+from numpy.testing import assert_array_equal, assert_array_almost_equal
+from numpy.testing import assert_almost_equal
+from sklearn import datasets, base
+from sklearn.utils import check_random_state
+from sklearn.utils.testing import assert_equal, assert_true
+from sklearn.utils.testing import assert_greater
+from sklearn.utils.testing import assert_raises
+from sklearn.externals import six
+from sklearn.utils.estimator_checks import check_estimator
+from sklearn.kernel_ridge import KRC
+
+# toy sample
+X = [[-2, -1], [-1, -1], [-1, -2], [1, 1], [1, 2], [2, 1]]
+Y = [1, 1, 1, 2, 2, 2]
+T = [[-1, -1], [2, 2], [3, 2]]
+true_result = [1, 2, 2]
+
+# also load the iris dataset
+iris = datasets.load_iris()
+rng = check_random_state(42)
+perm = rng.permutation(iris.target.size)
+iris.data = iris.data[perm]
+iris.target = iris.target[perm]
+
+
+def test_check_estimator():
```

Comment:
This should be run automatically in sklearn/tests/test_common.py:def test_non_meta_estimators

Reply:
You could however run `check_estimator(KernelRidgeClassifier(metric='precomputed'))`

Reply:
ok, so, test removed. Also, there already is a test for precomputed kernels.